docset/windows/storage/Disable-PhysicalDiskIdentification.md

```diff
@@ -52,8 +52,7 @@ This cmdlet requires a storage enclosure that supports SCSI Enclosure Services (
 
 ### Example 1: Disable the identification LED on all physical disks in a pool
 ```
-PS C:\>$StPool = (Get-StoragePool -FriendlyName "SpacePool")
-PS C:\> Disable-PhysicalDiskIndication -StoragePool $StPool
+PGet-PhysicalDisk -StoragePool $pool | Disable-PhysicalDiskIdentification
 ```
 
 The first command gets the storage poll named SpacePool by using the Get-StoragePool cmdlet.
```

Comment:
```suggestion
```

Reply:
@katy-pan - please confirm whether this line should be deleted, and if necessary, please commit the deletion.